src/components/ImagesCatalog.module.css

```
@@ -297,6 +284,18 @@
   overflow: hidden;
 }
 
+.securitySection :global(.tabs) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)